### PR TITLE
refactor: cleanup free drops component

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Gradual refactor. Related: https://github.com/kodadot/nft-gallery/issues/9575

#### Did your issue had any of the "$" label on it?

- [ ] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "version": "3.0.0",
   "private": true,
   "scripts": {
-    "dev": "nuxi prepare && cross-env NODE_OPTIONS=--openssl-legacy-provider PORT=9090 nuxi dev",
+    "dev": "pnpm prepare && cross-env NODE_OPTIONS=--openssl-legacy-provider PORT=9090 nuxi dev",
```
Comment (vikiival) [inline package.json:22]: Why?

**Comment by prury** [comment]
old drops got wrecked :smile: :
https://deploy-preview-10336--koda-canary.netlify.app/ahp/drops/chroma

image

File: package.json (hunk 1 above)
Comment (preschian) [reply package.json]: reverted

**Comment by preschian** [comment]
> old drops got wrecked 😄 :

aw 😅, updated

**Review by hassnian — APPROVED** [review]

File: utils/newsletter.ts [hunk 2]
```diff
@@ -28,38 +28,4 @@ async function subscribe(email: string) {
...
-    })
-
-    return data
-  } catch (e) {
-    throw new Error('[NEWSLETTER] Unable to resend confirmation email' + e)
-  }
-}
-
-export default { subscribe, getSubscription, resendConfirmationEmail }
+export default { subscribe }
```
Comment (Jarsen136) [inline utils/newsletter.ts:31]: It's safe to delete it too.

**Review by Jarsen136 — APPROVED** [review]

File: utils/newsletter.ts (hunk 2 above)
Comment (preschian) [reply utils/newsletter.ts]: thanks

**Comment by prury** [comment]
i'm trying to improve/fix the test

**Comment by prury** [comment]
@preschian removed the waitforloadstate on tests, since those commands do not truly work and added some timeouts.

waitforloadstate with any option could result in a milisecond timeout or a infinite one, unfortunately.

**Pull request merged**